src/settings-ui/Settings.UI/Views/ShortcutGuidePage.xaml

```diff
@@ -68,6 +68,7 @@
                         IsTabStop="True"
                         IsOpen="{x:Bind Mode=OneWay, Path=ViewModel.UseLegacyPressWinKeyBehavior}"
                         IsClosable="False"
+                        IsEnabled="{x:Bind Mode=OneWay, Path=ViewModel.IsEnabled}"
```

Comment:
`IsEnabled` doesn't work. You have to use `IsOpen`

Reply:
Uhm, I don't think so? There's already an `IsOpen` for this, which mean the same as _visibility". The problem is that when you turn off the entire module, this infobar won't change...

Reply:
But it doesn't work. I tried `IsEnabled` in my PR and it doesn't work. @niels9001 what do you think?

Reply:
@htcfreek @jay-o-way So there's no visual difference between Enabled/Disabled? Hmm, might be something to flag in the WinUI repo.

As a workaround now I'd set the Visibility or IsOpen instead?

Reply:
@niels9001
Do you like to open an issue in WinUI repo? I agree that this looks like a bug in WinUI.

Reply:
@niels9001

- can you explain the difference between `muxc:InfoBar` and `<InfoBar>`? According to [this page](https://docs.microsoft.com/nl-nl/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.infobar?view=windows-app-sdk-1.1#:~:text=Gets%20or%20sets%20a%20value%20indicating%20whether%20the%20user%20can%20interact%20with%20the%20control.) the **IsEnabled** property _should_ be possible, am I right?
- Like I already said - and you all can see - the is already an **IsOpen** property and it has a different purpose.

Reply:
> @niels9001
> 
> - can you explain the difference between `muxc:InfoBar` and `<InfoBar>`? According to [this page](https://docs.microsoft.com/nl-nl/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.infobar?view=windows-app-sdk-1.1#:~:text=Gets%20or%20sets%20a%20value%20indicating%20whether%20the%20user%20can%20interact%20with%20the%20control.) the **IsEnabled** property _should_ be possible, am I right?
> - Like I already said - and you all can see - the is already an **IsOpen** property and it has a different purpose.

We can remove the muxc tag: this was required for WinUI 2.x (in XAML Islands), but since we moved to WAS this can be removed and we can use e.g. <InfoBar../>

Yes, I am aware that there is an IsEnabled property and that it works (the InfoBar.Content gets disabled right?) but that doesn't mean that there is a Disabled state implemented as well that would update the style so it 'looks' disabled.

Apparently that is not the case, so we need to come up with a fallback.. leaving everything as is or hiding it?
I'd say this Disabled state would be nice to have for this control so that's why we could flag it in the WinUI repo, who owns the the control.

Reply:
https://github.com/microsoft/microsoft-ui-xaml/blob/main/dev/InfoBar/InfoBar.xaml

Looking at the XAML, there is no Disabled state :(. We could add our IsEnabledTextBlock in the Header/Title and Message - that should work. Wrapping the InfoBar in a ContentControl might work (but doubt it tbh).

Reply:
Other (muxc)InfoBar controls do set visual styles, but this single one is unique in that it has no content. I think it _should_ inherit from `SettingsGroup`, or we need to give it content.

Reply:
Yesh adding to the SettingsGroup might work.. but let's leave this out of scope for this PR now so we get this in?